### PR TITLE
must-gather: collect noobaa db pod info

### PR DESCRIPTION
**Pull request by rewantsoni**
1. Collect info for noobaa db pod and stateful set
2. get all resources with -o=wide in openshift-storage namespace

Bug: https://bugzilla.redhat.com/show_bug.cgi?id=1999405

Signed-off-by: Rewant Soni <resoni@redhat.com>

### CODE REVIEW

File: must-gather/collection-scripts/gather_noobaa_resources [hunk 1]
```diff
@@ -11,6 +11,9 @@ if [ "${BASE_COLLECTION_PATH}" = "" ]; then
     BASE_COLLECTION_PATH=$(pwd)
 fi
 
+# Make a global variable for namespace
+INSTALL_NAMESPACE=openshift-storage
```
Comment (jarrpa) [inline must-gather/collection-scripts/gather_noobaa_resources:15]: ...I'm just gonna say that I don't like this, we should be getting this from an env var or something, but I'm not going to ask for this to change. The noobaa gather script is a mess in many other ways and needs a broader refactor, which would prevent this.

**Review by jarrpa — CHANGES_REQUESTED** [review]

File: must-gather/collection-scripts/gather_namespaced_resources [hunk 2]
```diff
@@ -37,7 +37,7 @@ commands_get+=("subscription")
 commands_get+=("csv")
 commands_get+=("installplan")
 commands_get+=("events")
-commands_get+=("all")
+commands_get+=("all -o wide")
```
[the remark this answers is not in the document]
Comment (rewantsoni) [reply must-gather/collection-scripts/gather_namespaced_resources:40]: Okay, i will split it

File: must-gather/collection-scripts/gather_noobaa_resources (hunk 1 above)
Comment (rewantsoni) [reply must-gather/collection-scripts/gather_noobaa_resources:15]: Agreed, I will refactor the code and make it a bit cleaner after this PR

**Review by jarrpa — APPROVED** [review]
/lgtm

**Review by jarrpa — CHANGES_REQUESTED** [review]
Oops, did not mean to do that. I don't know why you resolved the discussion, but the commit is not yet split.

**Review by agarwal-mudit — APPROVED** [review]
/lgtm

**Pull request merged**